source/ns_cmdline.c

```diff
@@ -1892,3 +2101,47 @@ char *cmd_parameter_last(int argc, char *argv[])
     }
     return NULL;
 }
+static const char* strrevchr(const char* from, const char* to, const char c)
+{
+  if(from <= to) return 0;
+  while((from > to) && (*from == 0)) {
+    from--;
+  }
+  while((from > to) && (*from == c)) {
+    from--;
+  }
```

Comment:
Doesn't the above loop skip past the first block of the searched for characters if it's the last character before NUL in the range?

For example if the string between the pointers is "aaaabaabbb\0\0" and you search for 'b'.

Reply:
ah that's true - logic is really that it tries (fixing one bug soon) to find last character X but skip last ones if there is multiple--> is used to find "last" space....

`strrevchr("aaaabaabbb\0\0"  - 'b')` --> `bbb\0\0`